crates/edr_rpc_server/src/node/node_data.rs

```diff
@@ -0,0 +1,424 @@
+use std::sync::Arc;
+use std::time::{SystemTime, UNIX_EPOCH};
```

Comment:
Merge imports

Reply:
Done in [Merge imports](https://github.com/NomicFoundation/hardhat/pull/4539/commits/0e0a2f349ef63b628abbab1547476d3a621afd6d)